openapi/paths/aml-checks@{id}.yaml

```
@@ -0,0 +1,26 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - AML
+  summary: Retrieve a customer's AML check
+  operationId: GetAmlCheck
+  x-sdk-operation-name: get
+  description: |-
+    Retrieves the results of a customer's AML check.
+    Customer metadata and an array of matching AML hits is returned.
+  responses:
+    '200':
+      description: Customer's AML check retrieved.
```

nitpick: (related to Adam's comment on the summary)

```suggestion
      description: AML check retrieved.
```